src/base/bittorrent/session.cpp

```diff
@@ -2056,7 +2052,6 @@ LoadTorrentParams Session::initLoadTorrentParams(const AddTorrentParams &addTorr
     LoadTorrentParams loadTorrentParams;
 
     loadTorrentParams.name = addTorrentParams.name;
-    loadTorrentParams.tags = addTorrentParams.tags;
```

Comment:
Can you elaborate a little why this couldn't be done?

Reply:
We need to register all tags in Session. Torrent can properly have attached only registered tags.
Or do you ask something other?

Reply:
>We need to register all tags in Session. Torrent can properly have attached only registered tags.

I see.

Reply:
Also we need to filter out possible invalid tags, so we can't just assign them all at once.